### PR TITLE
feat(ActivityManager): Send notification about new submissions to circle memebers

### PR DESCRIPTION
**Pull request by susnux**
Follow up on #1496 and #1467

### CODE REVIEW

File: tests/Unit/Activity/ActivityManagerTest.php [hunk 1]
```diff
@@ -259,9 +265,12 @@ public function testPublishNewSharedSubmission(int $shareType, string $shareWith
 		$form->setOwnerId('formOwner');
 		$submitterId = 'submittingUser';
 
-		if ($groupUsers === null) {
-			$this->groupManager->expects($this->never())->method('get');
-		} else {
+		if ($shareType === IShare::TYPE_CIRCLE) {
+			$this->circlesService->expects($this->once())
+				->method('getCircleUsers')
+				->with('sharedCircles')
```
Comment (Chartman123) [inline tests/Unit/Activity/ActivityManagerTest.php:271]:
```suggestion
				->with('sharedCircle')
```

For groups we also only use the singular in the with-method (see line 241)

File: tests/Unit/Activity/ActivityManagerTest.php [hunk 2]
```diff
@@ -259,9 +265,12 @@ public function testPublishNewSharedSubmission(int $shareType, string $shareWith
...
+				->willReturn($groupUsers);
```
Comment (Chartman123) [inline tests/Unit/Activity/ActivityManagerTest.php:272]: I wouldn't expect `$groupUsers` here... Is this intended?

File: tests/Unit/Activity/ActivityManagerTest.php (hunk 1 above)
Comment (Chartman123) [reply tests/Unit/Activity/ActivityManagerTest.php:271]: This is also the reason for the failing PHPUnit checks

File: tests/Unit/Activity/ActivityManagerTest.php (hunk 2 above)
Comment (susnux) [reply tests/Unit/Activity/ActivityManagerTest.php]: renamed the argument so it does not confuse anymore ;-)

File: tests/Unit/Activity/ActivityManagerTest.php (hunk 1 above)
Comment (susnux) [reply tests/Unit/Activity/ActivityManagerTest.php]: Thank you, typo :see_no_evil:

**Review by Chartman123 — APPROVED** [review]

**Pull request merged**